File: Client/Assets/iTweenExamples/PunchAndStab/Scripts/Ticker.cs
using UnityEngine;
using System.Collections;

public class Ticker : MonoBehaviour{
	public AudioClip squeak;
	
	void Slam(int amount){
		iTween.PunchRotation(gameObject,iTween.Hash("y",amount,"time",2));	
		iTween.Stab(gameObject,iTween.Hash("audioclip",squeak,"pitch",Random.Range(.9f,1),"volume",Random.Range(.5f,1)));
	}
}

File: Client/Assets/Package/Pixelplacement/iTweenExamples/PunchAndStab/Scripts/Ticker.js
var squeak:AudioClip;

function Slam(amount:int){
	iTween.PunchRotation(gameObject,iTween.Hash("y",amount,"time",2));	
	iTween.Stab(gameObject,iTween.Hash("audioclip",squeak,"pitch",Random.Range(.9,1),"volume",Random.Range(.5,1)));
}

